Detectors/TRD/base/src/TrackletTransformer.cxx

```diff
@@ -96,22 +112,23 @@ float TrackletTransformer::calculateDy(int slope, double lorentzAngle, double dr
   // NOTE: check what drift height is used in calibration code to ensure consistency
   // NOTE: check sign convention of Lorentz angle
   // NOTE: confirm the direction in which vDrift is measured/determined. Is it in x or in direction of drift?
-  double lorentzCorrection = TMath::Tan(lorentzAngle) * mXAnode;
+  double lorentzCorrection = TMath::Tan(exb) * mXAnode;
 
   // assuming angle in Bailhache, fig. 4.17 would be positive in our calibration code
   double calibratedDy = rawDy - lorentzCorrection;
 
   return rawDy;
 }
 
-float TrackletTransformer::calibrateX(double x, double t0Correction)
+float TrackletTransformer::calibrateX(double x)
 {
-  return x += t0Correction;
+  // Hard-coded value will be replaced once t0 calibration is available from CCDB
+  float t0 = -0.279;
```

Comment:
Maybe call this t0Correction to not confuse it with the t0 of 4 time bins below?